### PR TITLE
[13.0][mrp_multi_level] add action parameter to  prepare mrp move data bom explosion

### PR DESCRIPTION
**Pull request by robinhli**

### CODE REVIEW

**Comment by robinhli** [comment]
> Hi @robinhli! Thank you very much for this contribution. As the addon you are improving does not have a declared maintainer, I take the opportunity to mention that you can consider adopting it. To do so, please read the [maintainer role description](https://odoo-community.org/page/maintainer-role), and, if interested, create a pull request to add your GitHub login to the `maintainers` key of the addon manifest.

I have add a new pull request to add me to maintainer, please approve.
https://github.com/OCA/manufacture/pull/698

File: mrp_multi_level/wizards/mrp_multi_level.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def _prepare_mrp_move_data_from_stock_move(
 
     @api.model
     def _prepare_planned_order_data(
-        self, product_mrp_area, qty, mrp_date_supply, mrp_action_date, name
+        self, product_mrp_area, qty, mrp_date_supply, mrp_action_date, name, values
```
Comment (dreispt) [inline mrp_multi_level/wizards/mrp_multi_level.py]: Breaking API change
```suggestion
        self, product_mrp_area, qty, mrp_date_supply, mrp_action_date, name, values=None
```

File: mrp_multi_level/wizards/mrp_multi_level.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def _prepare_planned_order_data(
 
     @api.model
     def _prepare_mrp_move_data_bom_explosion(
-        self, product, bomline, qty, mrp_date_demand_2, bom, name
+        self, product, bomline, qty, mrp_date_demand_2, bom, name, action
```
Comment (dreispt) [inline mrp_multi_level/wizards/mrp_multi_level.py]: Same here
```suggestion
        self, product, bomline, qty, mrp_date_demand_2, bom, name, action=None
```

**Review by dreispt — CHANGES_REQUESTED** [review]

**Comment by robinhli** [comment]
added the default value.

**Comment by dreispt** [comment]
Can you fix pre-commit please?

**Comment by robinhli** [comment]
> Can you fix pre-commit please?

sorry, I dont know how to change or find any problem. Could you please help, let me know how to change?

**Comment by robinhli** [comment]
the issue from precomit, don't know how to change. it seems this is from qc_trigger_line which is at another module. and seems not related my change. Could you please let me know what the issue is and how to change so my change can be merged?
Thanks!
@dreispt

************* Module quality_control_oca.models.qc_trigger_line
quality_control_oca/models/qc_trigger_line.py:55: [R7980(consider-merging-classes-inherited), QcTriggerProductCategoryLine] Consider merging classes inherited to "qc.trigger.line" from qc_trigger_line.py:81, qc_trigger_line.py:105.

**Comment by robinhli** [comment]
> Can you fix pre-commit please?

Dreispt, could you please let me know how to fix this? This is my first time to submit pull request.

**Pull request closed without merging**